student-library/src/main/java/tds/student/sql/data/ItemResponseScorable.java

```diff
@@ -172,7 +172,12 @@ public void setTestID(String testID) {
   }
 
   @Override
+  @JsonProperty ("ClientName")
   public String getClientName() {
     return _clientName;
   }
+
+  public void setClientName(String clientName) {
+    this._clientName = clientName;
+  }
```

Comment:
Related change, although the ItemResponseScorable has a "clientName" property, it was never being set before, which was causing issues during scoring.  I added a setter to allow setting the value during SAX parsing in Student.